actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -364,7 +364,7 @@ def POST
     # Returns the authorization header regardless of whether it was specified directly or through one of the
     # proxy alternatives.
     def authorization
-      get_header('HTTP_AUTHORIZATION')   ||
+      get_header("HTTP_AUTHORIZATION")   ||
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.